### PR TITLE
libcontainers-common: patch CVE-2021-43565

### PR DESCRIPTION
**Pull request by arc9693**
###### Merge Checklist  

**All** boxes should be checked before merging the PR *(just tick any boxes which don't apply to this PR)*
- [x] The toolchain has been rebuilt successfully (or no changes were made to it)
- [x] The toolchain/worker package manifests are up-to-date
- [x] Any updated packages successfully build (or no packages were changed)
- [x] Packages depending on static components modified in this PR (Golang, `*-static` subpackages, etc.) have had their `Release` tag incremented.
- [x] Package tests (%check section) have been verified with RUN_CHECK=y for existing SPEC files, or added to new SPEC files
- [x] All package sources are available
- [x] cgmanifest files are up-to-date and sorted (`./cgmanifest.json`, `./toolkit/scripts/toolchain/cgmanifest.json`, `.github/workflows/cgmanifest.json`)
- [x] LICENSE-MAP files are up-to-date (`./SPECS/LICENSES-AND-NOTICES/data/licenses.json`, `./SPECS/LICENSES-AND-NOTICES/LICENSES-MAP.md`, `./SPECS/LICENSES-AND-NOTICES/LICENSE-EXCEPTIONS.PHOTON`)
- [x] All source files have up-to-date hashes in the `*.signatures.json` files
- [x] `sudo make go-tidy-all` and `sudo make go-test-coverage` pass
- [x] Documentation has been updated to match any changes to the build system
- [x] If you are adding/removing a .spec file that has multiple-versions supported, please add [@microsoft/cbl-mariner-multi-package-reviewers](https://github.com/orgs/microsoft/teams/cbl-mariner-multi-package-reviewers) team as reviewer [(Eg. golang has 2 versions 1.18, 1.21+)](https://github.com/microsoft/azurelinux/tree/2.0/SPECS/golang)
- [x] Ready to merge

---

###### Summary 

What does the PR accomplish, why was it needed?
libcontainers-common: patch CVE-2021-43565 

###### Change Log  

- libcontainers-common: patch CVE-2021-43565 

###### Does this affect the toolchain?  

**NO**

###### Associated issues  

- #xxxx

###### Links to CVEs  
- https://nvd.nist.gov/vuln/detail/CVE-2021-43565

###### Test Methodology

- Pipeline build id: [608584](https://dev.azure.com/mariner-org/mariner/_build/results?buildId=608584&view=results)

### CODE REVIEW

**Review by mfrw — APPROVED** [review]

**Review by jslobodzian — APPROVED** [review]

File: SPECS/libcontainers-common/libcontainers-common.spec [hunk 1]
```diff
@@ -70,7 +71,9 @@ github.com/containers libraries, such as Buildah, CRI-O, Podman and Skopeo.
 
 %setup -q -T -D -b 9 -n common-%{commonver}
 %patch 0 -p1
+%patch 1 -p1
```
Comment (jslobodzian) [inline SPECS/libcontainers-common/libcontainers-common.spec:74]: This patch is being re-applied.  First application is on line 70.

**Review by jslobodzian — CHANGES_REQUESTED** [review]
Please verify the "patch1" redundancy.

File: SPECS/libcontainers-common/libcontainers-common.spec (hunk 1 above)
Comment (rikenm1) [reply SPECS/libcontainers-common/libcontainers-common.spec:74]: there was a merge conflict between two patches. both of them were patch 1. Made one of them 1 another 2. I will remove this line.

Comment (rikenm1) [reply SPECS/libcontainers-common/libcontainers-common.spec:74]: This is from my side.

**Review by rikenm1 — APPROVED** [review]

**Review by jslobodzian — APPROVED** [review]

**Pull request merged**